Source/SwiftLintFramework/Extensions/NSFileManager+SwiftLint.swift

```diff
@@ -9,8 +9,8 @@
 import Foundation
 
 extension NSFileManager {
-    internal func filesToLintAtPath(path: String) -> [String] {
-        let absolutePath = path.absolutePathStandardized()
+    internal func filesToLintAtPath(path: String, rootDirectory: String? = nil) -> [String] {
+        let absolutePath = (path.absolutePathRepresentation(rootDirectory ?? NSFileManager.defaultManager().currentDirectoryPath) as NSString).stringByStandardizingPath
```

Comment:
I didn't see that, I'll update that as well. What I did see was in [travis](https://travis-ci.org/realm/SwiftLint/jobs/154062676) was this:

```
/Users/travis/build/realm/SwiftLint/Tests/SwiftLintFramework/ConfigurationTests.swift:122:31: method does not override any method from its superclass

        private override func filesToLintAtPath(path: String) -> [String] {
                ~~~~~~~~      ^
```

Which I'm assuming came about from merging with master.